### ISSUE TITLE
Element label is missing required attribute role

### ISSUE THREAD

**Issue by olavivaino** [validator/validator#282]
HTML checker (https://validator.w3.org/nu/#textarea) states following validation error:
"Element label is missing required attribute role"

This validation error is either false positive or it incorrectly states validation issue. According to http://w3c.github.io/html/single-page.html#the-label-element element <label> does not have attribute "role". Also "role" is not a global attribute.

Steps to reproduce:
  1. copypaste following HTML to checker at https://validator.w3.org/nu/#textarea

        <!DOCTYPE html>
        <html>
        <head>
        <title>Test</title>
        </head>
        <body>
        <div>
	        <label for="date" data-target="#tooltip" data-toggle="collapse" aria-expanded="false" aria-controls="tooltip">Import date </label>
	        <input type="date" id="date">
         </div>
        <div id="tooltip">
        content
        </div>
        </body>
        </html>

  2. Observe error message

**Comment by sideshowbarker** [comment]
Thanks for catching this and taking time to report it. Fixed in https://checker.html5.org/ and https://validator.w3.org/nu/ but note that you’ll now see an error message about the `aria-expanded` attribute, which is not allowed for the `label` element.

**Comment by jaideraf** [comment]
I think this bug has returned, now with the `input` element. Paste this code [here](https://validator.w3.org/nu/#textarea):

```
<!DOCTYPE html>
<html lang="pt-BR">
<head>
<title>Test</title>
</head>
<body>
<div>
<input class="form-check-input" type="checkbox" id="inlineFormCheck" data-bs-toggle="collapse" data-bs-target="#extra" aria-expanded="false" title="Exibe ou esconde o operador booleano">
 </div>
<div id="tooltip">
content
</div>
</body>
</html>
```
You will get:

`Error: Element [input](https://html.spec.whatwg.org/multipage/#the-input-element) is missing required attribute role.`

**Comment by jaideraf** [comment]
Well, this is not what I am reporting anyway... What I am trying to say is about `required attribute role` on `input` tag. `role` is not a `required attribute` for `input`.

**Comment by sebastian-kopiczko** [comment]
Can we expect any update on this since it's sholudn't be considered as a bug?
